src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure implements AreaCalculator, Drawable{
```

Comment:
```suggestion
public class Circle extends Figure {
```
можна фігурою імплементувати ці два інтерфейси і тоді, вони за замувчуванням перейдуть усім наслідникам